### PR TITLE
RIA-305 Hello World

### PR DESCRIPTION
**Pull request by fantasticjamieburns**

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+sonar.projectName=IA :: Immigration & Asylum appeal frontend
+sonar.projectKey=IAAPFR
+sonar.sources=.
+sonar.exclusions=node_modules/**,test/**,app/assets/**,public/**,govuk_modules/**,app-insights.js,Gruntfile.js,server.js,app.js,app/routes.js
```
Comment (davet1985) [inline sonar-project.properties]: I've amended this list, as it contains some files you might not have anymore e.g. `Gruntfile.js`

**Comment by davet1985** [comment]
Is there a specific reason for using codecept for the smoke tests and not the functional tests?

File: sonar-project.properties (hunk 1 above)
Comment (fantasticjamieburns) [reply sonar-project.properties]: Updated 👍

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+node_modules/
+public/
+govuk_modules/
```
Comment (davet1985) [inline .eslintignore]: Minor, but I don't think this directory exists.

**Pull request merged**